third_party/bigframes_vendored/pandas/core/common.py

```diff
@@ -0,0 +1,44 @@
+# Contains code from https://github.com/pandas-dev/pandas/blob/main/pandas/core/common.py
+from __future__ import annotations
+
+from typing import Callable, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from third_party.bigframes_vendored.pandas.pandas._typing import T
```

Comment:
This needs to be synced with `main` and updated.

```suggestion
    from bigframes_vendored.pandas.pandas._typing import T
```

Reply:
done